### ISSUE TITLE
Error when solving conda env with basilisk: python 3.7.7 missing

### ISSUE THREAD

**Issue by m-jahn** [crisprVerse/crisprScore#20]
Dear crisprVerse developers,
I recently got an error when trying to add `ruleset3` on-target scores with the `crisprScore` package.
Apparently basilisk is not able to solve the desired environment for ruleset 3, due to the missing python 3.7.7. Is there a workaround, like defining a different python version?

Thanks for your help.

```
[runCrisprBowtie] Using NA 
[runCrisprBowtie] Searching for SpCas9 protospacers 
# reads processed: 5581
# reads with at least one alignment: 5581 (100.00%)
# reads that failed to align: 0 (0.00%)
Reported 10034 alignments

[addOnTargetScores] Adding ruleset3 scores. 

+ user/.cache/R/basilisk/1.14.1/0/bin/conda create --yes --prefix user/.cache/R/basilisk/1.14.1/crisprScore/1.6.0/rs3__basilisk 'python=3.7.7' --quiet -c bioconda -c conda-forge
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed

PackagesNotFoundError: The following packages are not available from current channels:

  - python=3.7.7

Current channels:

  - https://conda.anaconda.org/bioconda/linux-64
  - https://conda.anaconda.org/bioconda/noarch
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


Error: Error creating conda environment 'user/.cache/R/basilisk/1.14.1/crisprScore/1.6.0/rs3__basilisk' [exit code 1]
```

**Comment by m-jahn** [comment]
After some investigation I found that `python=3.7.7` is indeed no longer available on the `conda-forge` channel. I also found that this was fixed in one of the latest versions of crisprscore (Version: 1.9.1, commit becdae7b9033ee086d87e814a27100ca96afd1b3).

The problem just persists for me and my pipeline https://github.com/MPUSP/snakemake-crispr-guides, because the latest `crisprscore` package is not available on `bioconda` yet. The latest _available_ version is 1.6.0.
